lib/pbench/server/database/alembic/env.py

```diff
@@ -0,0 +1,112 @@
+"""
+This file is auto generated when we run `alembic init alembic` but modified according to our needs.
```

Comment:
I'm confused...is this file auto-generated or not?  I understand the concept of taking an auto-generated file and modifying it, but the problem with that is, if the generator is ever re-invoked, then we lose our modifications.  What prevents that from happening, here?

Moreover, what if we upgrade to a new version of alembic which generates additional stuff (or generates this file differently)?

This seems like a risky appoach.  Wouldn't it be better to wrap the generated file inside one that we control, or something similar which insulates us a bit?

Reply:
That's a fair point I will create a ticket to track this issue

Reply:
The fact that it won't be overwritten is reassuring on one level -- we won't lose your hard work, here -- but, if it means that it breaks the build (because the command fails) that will be annoying, and, worse, if we cannot generate a new file when it's appropriate, that's a whole 'nother set of problems.

So, I still think we would benefit from a different approach here.